drainer/translator/mysql.go

```diff
@@ -31,6 +31,8 @@ import (
 
 const implicitColID = -1
 
+var destDBType = "tidb"
```

Comment:
Can we use an iota for this variable? I'm affray that string type is not accurate and may affect the efficiency.

Reply:
```suggestion
type destDBType int
var (
    mysql = iota
    oracle
)
```

Reply:
OK，good idea